cms/envs/common.py

```diff
@@ -1513,6 +1513,9 @@
     'openedx.core.djangoapps.content.learning_sequences.apps.LearningSequencesConfig',
 
     'ratelimitbackend',
+
+    # Event Tracking library, xAPI
+    'eventtracking',
```

Comment:
@ reviewer: Is there some logic to how apps are added to Installed_apps? It does not seem to be alphabetically.